applications/tuxedo_parachain.md

```diff
@@ -0,0 +1,194 @@
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Joshy Orndorff https://github.com/JoshOrndorff
+- Off Narrative Labs https://github.com/Off-Narrative-Labs
+
+### Contact
+
+- **Contact Name:** Joshy Orndorff 
+- **Contact Email:** joshyorndorff at proton dot me
+- **Registered Address:** To be provided in the invoices
+
+### Team's experience
+
+Joshy entered the Substrate ecosystem in 2019 as part of the Substrate Developer Hub team. There he created and hosted the weekly Substrate Seminar, and contributed significantly to the Substrate Recipes. In 2020, he moved to the Moonbeam team where he was a core developer. While at Moonbeam, Joshy wrote the [Nimbus consensus engine](https://github.com/PureStake/nimbus/) which is used in several production parachains, and helped pioneer the technique whereby EVM contracts can interact with native Substrate pallets. In 2022, Joshy began contributing to the Polkadot Blockchain Academy, teaching in all three cohorts: Cambridge, Buenos Aires, and Berkeley.
+
+Joshy met Andrew Burger in Cambridge in 2022 at the first Polkadot Blockchain Academy where Andrew, a student at the time, chose to implement a UTXO Runtime as his final project. Together they revised the UTXO assignment and taught it together at the next PBA. A few months later they worked together on the first Tuxedo grant.
+
+Andrew will not be contributing to this grant directly (although he is still an enthusiastic open source contributor to Tuxedo) due to conflicting employment elsewhere.
+
+### Team Code Repos
+
+* Main Tuxedo Repo - https://github.com/Off-Narrative-Labs/Tuxedo
+* Tuxedo Tutorial - https://github.com/Off-Narrative-Labs/Tuxedo-Order-Book-Dex-Tutorial/
+* Nimbus Consensus (Prior Work) - https://github.com/moonbeam-foundation/nimbus
+* Blockchain From Scratch (Educational) https://github.com/JoshOrndorff/blockchain-from-scratch
+
+## Development Status :open_book:
+
+As mentioned above, the team has done previous work on [Tuxedo](https://github.com/Off-Narrative-Labs/Tuxedo/tree/main).
+
+As we mentioned in our previous grant our next tasks are parachain support (this grant's topic) and zero knowledge UTXOs.
+
+Of course there are also some lingering leftovers in our issue queue mostly centered around the wallet.
+We do at some point want to make the wallet more generalizable to be able to craft any generic transaction based on whichever tuxedo runtime is defined.
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+
+- **Total Estimated Duration:** 12 weeks
+- **Full-Time Equivalent (FTE):**  1 FTE 
+- **Total Costs:** $48,000 (USD)
+
+### Milestone 1 — Rewrite Pallet Parachain System to UTXO model 
+
+- **Estimated duration:** 6 weeks
+- **FTE:**  1
+- **Costs:** 24,000 USD
+
+The bulk of the logic for handling communication with the relay chain lives in pallet parachain system, and consequently, this is the largest milestone in this grant proposal.

```

Comment:
```suggestion
The bulk of the logic for handling communication with the relay chain lives in the pallet parachain system, and consequently, this is the largest milestone in this grant proposal.

```

I didn't really get what you mean here; "parachain system" is the name of the pallet? Or do you mean "pallet parachain system" as a single term? If so, what is the definition of that system?

Reply:
The name of the pallet is "Pallet Parachain System" or maybe just "Parachain System". I think this is the common way to refer to them, right? Like "Pallet Balances" or "Pallet Staking".

In any case, this is the thing I'm referring to, and this link is already included previously in the application. https://paritytech.github.io/cumulus/cumulus_pallet_parachain_system/index.html

Reply:
```suggestion
The bulk of the logic for handling communication with the relay chain lives in [pallet parachain system](https://paritytech.github.io/cumulus/cumulus_pallet_parachain_system/index.html), and consequently, this is the largest milestone in this grant proposal.

```

Reply:
Great, thanks for the clarification.